csm_web/frontend/src/components/course/CreateSectionModal.tsx

```diff
@@ -104,6 +110,13 @@ export const CreateSectionModal = ({ courseId, closeModal, reloadSections }: Cre
     });
   };
 
+  const handleSelect = (newSelections: any) => {
+    const labelIDs = newSelections.map((label: any) => label.value);
+    setLabels(labelIDs);
+  };
+
+  const animatedComponents = makeAnimated();
```

Comment:
What does this do? A comment here explaining a little bit would be useful.

Reply:
done !